resources_pages/contributing_blog.md

```diff
@@ -1,16 +1,60 @@
 ---
 layout: page
-title: Contributing to the MDS blog
+title: Contributing to the MDS Blog
 ---
 
-To contribute to the [MDS blog](https://ubc-mds.github.io/), follow these steps to work with the blog's Editor, which is currently Varada Kolhatkar (`@kvarada`):
-
-1. Contact the Editor and agree on the basic outline of the post. 
-2. Fork [the repo that houses this site](https://github.com/ubc-mds/ubc-mds.github.io) and give the Editor write access to the fork.
-3. Create a new file in the `_posts` directory, write a draft, and push it to your fork. 
-    - For eventual social media posting it's best to include a `share-img`; for example, see the 3rd line [here](https://raw.githubusercontent.com/UBC-MDS/UBC-MDS.github.io/master/_posts/2019-08-22-project-courses.md).
-    - Please also add a byline at the bottom with your name and whatever you want to say about yourself.
-    - Optionally, consider adding a line break after each line of text. This will allow the diffs to show up much better while the post is being edited.
-4. Submit a pull request from your fork to the blog repo (but be careful not to send it to the [parent of the blog repo](https://github.com/daattali/beautiful-jekyll), which can happen by accident).
-5. Iterate back and forth with the Editor until the post is ready.
-6. The Editor will merge the PR and the post will automatically appear on the MDS blog.
+Want to share your insights with the MDS community? Contributing to the [MDS blog](https://ubc-mds.github.io/) is easy and fun! Just follow these steps to get your blog post published with the help of our Editor, Varada Kolhatkar (`@kvarada`):
+
+
+### 1. Reach Out
+
+First things first—get in touch with the Editor to discuss your blog idea and how you'd like to contribute. We're excited to hear what you have in mind!
```

Comment:
It would be good to add an email for the Editor. Otherwise, they won't know who to reach out to.

Reply:
Should I use your email or ours?

Reply:
You can put my email for now. In future, it would be nice to create and use a generic MDS course coordinator email.

Reply:
Great idea! I'll discuss that with Betty and Zac in the future.